.github/workflows/master_std_ui_experimental.yml

```diff
@@ -5,36 +5,30 @@ on:
   # Variables to set when calling this reusable workflow
     inputs:
       browser:
-        description: Web browser to test
+        description: "BROWSER where the test will run"
         required: true
         type: string
         default: chrome
       cypress_docker:
-        description: Cypress docker image to use
+        description: "CYPRESS DOCKER IMAGE (ensure compatibilty with selected browser)."
         required: true
         type: string
         default: cypress/included:10.11.0 
       cypress_spec:
-        description: Which Cypress test to execute
-        required: true
-        type: string
-        default: ''
-      # grep:
-      #   description: Select spefic test(s)
-      #   required: false
-      #   default: ''
-      docker_options:
```

Comment:
This is used later and also needed for firefox. So please either get rid of that completely and support chrome only (so no browsers) or get it back.

Reply:
thanks.
will return it back then